server/services/sentence-plan/goalService.ts

```diff
@@ -17,6 +17,7 @@ export default class GoalService {
     return restClient.put<Goal>({ path: `/goals/${goalUuid}`, data: goal })
   }
 
+  // rename this to updateGoalStatus
```

Comment:
Does this need renaming?

Reply:
Yes. I should have added a TODO note! I'll do this.

Reply:
done in 7c00fc4719405bbc06035ef65f855887cd24ab34